docs/src/Buttons.elm

```diff
@@ -1,249 +1,189 @@
 module Buttons exposing (stories)
 
-import Element as El
-import Element.Background as Background
+import Element
 import Msg as Msg
-import UI.Button as B
-import UI.Colors as Colors
-import UI.Icons as Icons
+import UI.Button as Btn
+import UI.Icons as Ico
 import UIExplorer exposing (storiesOf)
 import Utils exposing (story, storyList)
 
 
-baseButton : B.Button Msg.Msg
-baseButton =
-    B.button Msg.NoOp
-        |> B.withText "Click here"
-
-
-stories =
+stories renderConfig =
     storiesOf
         "Buttons"
-        [ primaryStory
-        , successStory
-        , warningStory
-        , dangerStory
-        , disabledButtonStory
-        , toggleStory
-        , fullWidthStory
-        , linkStory
+        [ primaryStory renderConfig
```

Comment:
- We should remove the outline story
- We should add a functional toggle story, so maybe it's better to remove toggle from all the stories and create a specific toggle story but with a message and model.